issue-creator/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: CronJob
```

Comment:
Switched back to prow periodic job by assigning users with `/assign` commands instead of specifying assignees in the issue creation request. This will let Prow handle the collaborator validation and if the assignee is invalid, the user will be mentioned instead of failing to create the issue.